thanks-bot/bot.js

```diff
@@ -1,35 +1,57 @@
-var Discord = require('discord.io');
-var logger = require('winston');
-var auth = require('./auth.json');
+const Discord = require('discord.js')
+const logger = require('winston')
+const auth = require('../auth.json')
 
 // Configure logger settings
-logger.remove(logger.transports.Console);
+logger.remove(logger.transports.Console)
 logger.add(logger.transports.Console, {
-    colorize: true
-});
-
-logger.level = 'debug';
+  colorize: true
+})
+logger.level = 'debug'
 
 // Initialize Discord Bot
-var bot = new Discord.Client({
-   token: auth.token,
-   autorun: true,
-});
+const bot = new Discord.Client()
 
 bot.on('ready', function (evt) {
```

Comment:
[According to the documentation](https://discord.js.org/#/docs/main/stable/class/Client?scrollTo=e-ready), there are no arguments for this function, i.e.
```suggestion
bot.on('ready', function () {
```
or you can make
```suggestion
bot.on('ready', () => {
```